lib/scope.js

```diff
@@ -85,9 +85,10 @@ ScopeDefinition.prototype.related = function(receiver, scopeParams, condOrRefres
     }
   }
   cb = cb || utils.createPromiseCallback();
-
-  if (!self.__cachedRelations || self.__cachedRelations[name] === undefined ||
-      actualRefresh) {
+  const _conditions = !self.__cachedRelations ||
```

Comment:
Please use a more descriptive name instead of `_conditions`, e.g. `refreshIsNeeded` or `shouldFetchFreshData`.